examples/advanced/amplify/figs/plot_training_curves.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
```

Comment:
do we need this line?

Reply:
no harm keeping it. Allows you to run the script directly without typing "python".